mithril-common/Cargo.toml

```diff
@@ -34,6 +34,8 @@ kes-summed-ed25519 = { version = "0.2.1", features = [
     "sk_clone_enabled",
 ] }
 nom = "7.1.3"
+pallas-network = { git = "https://github.com/falcucci/pallas.git", branch = "test/add-epoch-no" }
+pallas-codec = { git = "https://github.com/falcucci/pallas.git", branch = "test/add-epoch-no" }
```

Comment:
why do we need changes from `test/add-epoch-no`? what's the blocker in pallas `main` branch code?

Reply:
yep we do need [this](https://github.com/txpipe/pallas/pull/335) PR in specific and then rollback to our original pallas repo.